packages/parachains-impl/bifrost/libs/formats/currency.ts

```diff
@@ -1,7 +1,9 @@
 import type { ZenlinkProtocolPrimitivesAssetId } from '@zenlink-interface/format'
 import { addressToZenlinkAssetId, zenlinkAssetIdToAddress } from '@zenlink-interface/format'
+import { ParachainId } from '@zenlink-interface/chain'
 import type { NodePrimitivesCurrency } from '../../types'
 import { pairAddressToAssets } from '../constants'
+import { parseNodePrimitivesCurrency as amplitudeParseNodePrimitivesCurrency } from '../../../amplitude/libs/formats/currency'
```

Comment:
It's better to impl it on your own chain impl, not include in another chain impl

Reply:
As far as I can tell this is necessary, because, let's say you are connected to the 'Amplitude' network in the UI and try to navigate to the details for a Bifrost pool. You will encounter the following error because the app will try to parse the currencies with the Amplitude implementation although it's a bifrost pool and should use the bifrost parsing instead. The same is necessary vice versa. That's why I added [this](https://github.com/pendulum-chain/zenlink-interface/blob/7b6dc630ecd0516ebb04f02341da18dbe7dc654c/packages/parachains-impl/amplitude/libs/formats/currency.ts#L44-L45) and [this](https://github.com/pendulum-chain/zenlink-interface/blob/7b6dc630ecd0516ebb04f02341da18dbe7dc654c/packages/parachains-impl/bifrost/libs/formats/currency.ts#L92-L93) statement.

<img width="1144" alt="image" src="https://github.com/zenlinkpro/zenlink-interface/assets/6690623/42aac698-c2a3-4109-9912-7f407bcb76f9">